starky/src/stark.rs

```diff
@@ -67,10 +67,11 @@ pub trait Stark<F: RichField + Extendable<D>, const D: usize>: Sync {
         zeta: F::Extension,
         g: F::Extension,
         rate_bits: usize,
+        num_challenges: usize,
     ) -> FriInstanceInfo<F, D> {
```

Comment:
Let's add a TODO to incorporate `permutation_zs` in this method, now that we're adding some `permutation_zs` code elsewhere

Reply:
It's already on line 65 I think.